exe/manageiq-password

```diff
@@ -41,7 +41,24 @@ if [ ! -r "$keyfile" ]; then
 	exit 1
 fi
 
-algorithm="$(grep :algorithm: "$keyfile" | cut -d ' ' -f 2)"
+extractFromKeyfile() {
+	grep :$1: "$keyfile" | cut -d ' ' -f 2
+}
+
+extractAlgorithm() {
+	extractFromKeyfile "algorithm"
+}
+
+extractKey() {
+	extractFromKeyfile "key" | base64 -d | xxd -p -c256
+}
+
+extractIv() {
+	iv=$(extractFromKeyfile "iv" | base64 -d | xxd -p -c256)
+	echo -n "${iv:-"00000000000000000000000000000000"}"
```

Comment:
Wish there was a way to do this in bash without an intermediate var, but I can't figure out how.